projects/valtimo/process-link/src/lib/components/select-form-flow/select-form-flow.component.ts

```diff
@@ -140,6 +140,11 @@ export class SelectFormFlowComponent implements OnInit, OnDestroy {
       const updateProcessLinkRequest: FormFlowProcessLinkUpdateRequestDto = {
         id: selectedProcessLink.id,
         formFlowDefinitionId: this.selectedFormFlowDefinition.id,
+        ...(this.taskPanelToggle &&
+          this.isUserTask$.getValue() && {
+            formDisplayType: this.formDisplayValue,
+          }),
+        ...(this.taskPanelToggle && this.isUserTask$.getValue() && {formSize: this.formSizeValue}),
```

Comment:
Rewrite this to use rxjs operators instead of getValue from the behavior subject